pkg/controller/daemonset/daemonset_controller.go

```diff
@@ -596,24 +596,29 @@ func (dsc *ReconcileDaemonSet) manage(ds *appsv1alpha1.DaemonSet, hash string) (
 		return delay, fmt.Errorf("couldn't get nodeList %v", err)
 	}
 
-	var nodesNeedingDaemonPods, podsToDelete []string
+	var nodesNeedingDaemonPods, nodesDesireScheduled, podsToDelete []string
```

Comment:
can this `nodesDesireScheduled` be an integer?

Reply:
The last two comments have been fixed in the most recent commit :)